king_phisher/client/dialogs/configuration.py

```diff
@@ -221,6 +223,19 @@ def _configure_settings_plugins(self):
 		for plugin_klass in plugin_klasses:
 			self._configure_settings_plugin_options(plugin_klass)
 
+	def _configure_settings_proxy(self):
+		formatted_proxy_url = urllib.parse.urlparse(self.config['proxy.url'])
```

Comment:
This is also crashing for me when `self.config['proxy.url']` is None. To reproduce it just start King Phisher without a proxy configuration then open `Edit > Preferences`.

This is the stack trace:
```
Error Type: builtins.TypeError
Error Details: TypeError('Must be string, not bytes')
Error UID: 592845cb-a8a3-4324-84bb-8e84aaa41458
RPC Error: N/A (Not an RPC error)
King Phisher Version: 1.14.0-beta1 (rev: dad7205fa100)
Platform Version: Linux: Fedora 30 Thirty
Python Version: 3.7.3
Gtk Version: 3.24.8
Timezone: America/New_York

Thread Information:
===================
 => MainThread (alive=True daemon=False)
    pydevd.Writer (alive=True daemon=True)
    pydevd.Reader (alive=True daemon=True)
    pydevd.CommandThread (alive=True daemon=True)
    Thread-23 (alive=True daemon=True)
    Thread-20 (alive=True daemon=False)
    RPCAsyncWorker (alive=True daemon=False)
    Thread-27 (alive=True daemon=True)
    Thread-28 (alive=True daemon=True)
    Dummy-56 (alive=True daemon=True)
    Dummy-57 (alive=True daemon=True)

Stack Trace:
============
Traceback (most recent call last):
  File "/home/steiner/Repositories/king-phisher/king_phisher/client/windows/main.py", line 120, in signal_activate_edit_preferences
    self.application.show_preferences()
  File "/home/steiner/Repositories/king-phisher/king_phisher/client/application.py", line 779, in show_preferences
    if dialog.interact() != Gtk.ResponseType.CANCEL:
  File "/home/steiner/Repositories/king-phisher/king_phisher/client/dialogs/configuration.py", line 277, in interact
    self._configure_settings_proxy()
  File "/home/steiner/Repositories/king-phisher/king_phisher/client/dialogs/configuration.py", line 237, in _configure_settings_proxy
    self.gtk_builder_get('entry_proxy_url').set_text(proxy_url)
TypeError: Must be string, not bytes
```

The issue is that when `self.config['proxy.url']` is `None`, the members of the `ParseResult` are bytes, which is some straight up crazy talk but hey I didn't write this API. :man_shrugging: 

```
In [2]: urllib.parse.urlparse('test')                                                                                                                                                                               
Out[2]: ParseResult(scheme='', netloc='', path='test', params='', query='', fragment='')

In [3]: urllib.parse.urlparse(None)                                                                                                                                                                                 
Out[3]: ParseResultBytes(scheme=b'', netloc=b'', path=b'', params=b'', query=b'', fragment=b'')  
```

Anyways if `self.config['proxy.url']` is None, you can safely skip setting the entries since they'll default to blank anyways.